resources/public/api/conf/2.0/schemas/historic_uk_non_fhl_period_summary_retrieve/response.json

```diff
@@ -0,0 +1,161 @@
+{
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
+  "type": "object",
+  "properties": {
+    "fromDate": {
+      "type": "string",
+      "description": "The first day that the income and expenses period summary covers. Must conform to the format YYYY-MM-DD",
+      "example": "2020-01-01"
+    },
+    "toDate": {
+      "type": "string",
+      "description": "The last day that the income and expenses period summary covers. Must conform to the format YYYY-MM-DD",
+      "example": "2020-01-31"
+    },
+    "income": {
+      "type": "object",
+      "description": "Object containing the details about income",
+      "properties": {
+        "periodAmount": {
+          "type": "number",
+          "description": "Total rents from property, ground rents and rent charges but not rent-a-room.\nThe value must be between 0 and 99999999999.99 up to 2 decimal places.",
+          "examples": "5000.99"
+        },
+        "premiumsOfLeaseGrant": {
+          "type": "number",
+          "description": "Total rents from property, ground rents and rent charges but not rent-a-room.\nThe value must be between 0 and 99999999999.99 up to 2 decimal places.",
+          "examples": "5000.99"
+        },
+        "reversePremiums": {
+          "type": "number",
+          "description": "Total rents from property, ground rents and rent charges but not rent-a-room.\nThe value must be between 0 and 99999999999.99 up to 2 decimal places.",
+          "examples": "5000.99"
+        },
+        "otherIncome": {
+          "type": "number",
+          "description": "Total rents from property, ground rents and rent charges but not rent-a-room.\nThe value must be between 0 and 99999999999.99 up to 2 decimal places.",
+          "examples": "5000.99"
+        },
+        "taxDeducted": {
+          "type": "number",
+          "description": "Tax already deducted from the rental income.\nThe value must be between 0 and 99999999999.99 up to 2 decimal places.",
+          "examples": "5000.99"
+        },
+        "rentARoom": {
+          "type": "object",
+          "description": "Objects holding rents received for the period",
+          "properties": {
+            "rentsReceived": {
+              "type": "number",
+              "description": "Total rents received from properties.\nThe value must be between 0 and 99999999999.99 up to 2 decimal places.",
+              "examples": "5000.99"
+            }
+          }
+        }
+      }
+    },
+    "expenses": {
+      "type": "object",
+      "description": "Object holding expenses for the period",
+      "properties": {
+        "premisesRunningCosts": {
+          "type": "number",
+          "description": "Rent, rates, insurance, ground rents and other costs.\nThe value must be between 0 and 99999999999.99 up to 2 decimal places.",
+          "examples": "5000.99"
```

Comment:
None of the examples in the expenses object are displaying either, again probably because "examples" needs to be singular.